Sources/XCBuildSupport/XcodeBuildSystem.swift

```diff
@@ -218,7 +218,8 @@ extension BuildConfiguration {
 extension PIFBuilderParameters {
     public init(_ buildParameters: BuildParameters) {
         self.init(
-            enableTestDiscovery: buildParameters.enableTestDiscovery,
+            // 👀 not sure
```

Comment:
👀

Reply:
This was introduced in #2918, removing it will break testing the Swift toolchain itself.

Reply:
I don't have any history here at all.  I'll take a look at the PIF generation and see how it works.